### PR TITLE
[WIP] Add raw_data_parent_id to normalised events

### PR DESCRIPTION
**Pull request by rhiaro**

### CODE REVIEW

**Comment by rhiaro** [comment]
[Make deletes cascade from source to normalised data](https://trello.com/c/2nIw5IhG/23-make-deletes-cascade-from-source-to-normalised-data)

File: src/lib/database-migrations/010-normalised-parent-id.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+-- If a normalised event was derived from two raw data objects, parentId stores the raw_data_id of the highest in the hierarchy
+--
+ALTER TABLE normalised_data
+ADD raw_data_parent_id TEXT NULL DEFAULT NULL,
```
Comment (odscjames) [inline src/lib/database-migrations/010-normalised-parent-id.sql:4]: This should match raw_data(id) so BIGSERIAL

Comment (odscjames) [reply src/lib/database-migrations/010-normalised-parent-id.sql]: Woops, BIGINT

**Comment by odscjames** [comment]
We should be using Bigints not Bigserial for foreign keys - I took this opportunity to fix an earlier bug!

**Comment by odscjames** [comment]
Tested by scrapping my DB and starting again - ran 2000 normalised items

**Pull request merged**